circle.yml

```diff
@@ -0,0 +1,19 @@
+machine:
+  services:
+    - docker
+
+test:
```

Comment:
I didn't know that could be tested at image-build time.  Originally that was the main test I was considering.  I'll need to ask @terryhorner how that would be accomplished.